sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchAudience.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents;
+
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.logging.ClientLogger;
+
+/**
+ * Cloud audiences available for Search.
+ */
+public final class SearchAudience {
```

Comment:
We have modeled this as an expandable string enum in other services. See [ContainerRegistryAudience](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/ContainerRegistryAudience.java). We should use the same pattern here.